kpconv_torch/utils/trainer.py

```diff
@@ -99,8 +99,9 @@ def train(self, net, training_loader, val_loader, config):
 
         if config.saving:
             # Training log file
-            with open(self.train_save_path / "training.txt", "w") as fobj:
-                fobj.write("epochs steps out_loss offset_loss train_accuracy time\n")
+            if not (self.train_save_path / "training.txt").exists():
```

Comment:
Why is the file only filled with values when it does not already exists? Should it be deleted and recreated when it already exists?

Reply:
Nope, here we are writing the file header. I did the change because the header must be written only on the first training run (i.e. when the file does not exist).

Actually this is related to another change of this PR: the behavior on `master` is such that we write only the new epoch metrics (by erasing the metrics generated during the previous training run). Now we **append** the training metrics to the existing ones, when the training is run from an existing checkpoint.